exasol_script_languages_developer_sandbox/lib/run_export_vm.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import traceback
+from sys import stderr
+from typing import Tuple
+
+from exasol_script_languages_developer_sandbox.lib.asset_id import AssetId
+from exasol_script_languages_developer_sandbox.lib.aws_access import AwsAccess
+from exasol_script_languages_developer_sandbox.lib.cf_stack import find_ec2_instance_in_cf_stack
+from exasol_script_languages_developer_sandbox.lib.print_assets import print_assets
+from exasol_script_languages_developer_sandbox.lib.render_template import render_template
+from exasol_script_languages_developer_sandbox.lib.vm_disk_image_format import VmDiskImageFormat
+from exasol_script_languages_developer_sandbox.lib.vm_slc_bucket import find_vm_bucket, find_vm_import_role
+
+def export_vm(aws_access: AwsAccess,
+              instance_id: str,
+              vm_image_formats: Tuple[str, ...],
+              asset_id: AssetId) -> None:
+    vm_bucket = find_vm_bucket(aws_access)
+    vmimport_role = find_vm_import_role(aws_access)
+    tag_value = asset_id.tag_value
+    bucket_prefix = asset_id.bucket_prefix
+    ami_name = asset_id.ami_name
+    has_errors = False
+    try:
+        try:
+            logging.info(f"create ami with name '{ami_name}' and tag(s) '{tag_value}'")
+            ami_id = "ami-055e106c605a2fa41"#aws_access.create_image_from_ec2_instance(instance_id, name=ami_name, tag_value=tag_value,
```

Comment:
??? test value

Reply:
Yes, the PR was not ready yet. It's fixed now.